ADD_SAMPLE_INTO_DEV_KIT.md

```diff
@@ -0,0 +1,72 @@
+# How to add samples to sample gallery of Office Add-ins Development Kit
+
+## Configure the sample for the Dev Kit
+
+The Dev Kit sample gallery uses the [samples-config.json](./.config/sample-config.json) file in this repo's `main` branch. It uses the sample configuration ID find the files, downloads a .zip file of the sample project from this repo, and unzips that file in the designated path.
+```
+{
+    "filterOptions": {
+        "capabilities": [
+        ],
+        "languages": [
+        ],
+        "technologies": [
+        ]
+    },
+    "samples": [
+        {
+            "id": "excel-get-started-with-dev-kit",
+            "onboardDate": "2024-11-01",
+            "title": "Excel - Get Started with the Dev Kit",
+            "description": "This sample demonstrates how to create a basic Excel add-in with a task pane that uses JavaScript.",
+            "types": [
+                "Excel"
+            ],
+            "tags": [
+                "JavaScript",
+                "Get Started",
+                "Excel"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "suggested": false
+        },
+    ]
+}
+```
+
+## Branch
+
+For new samples, please use `dev` as the target branch for pull requests. The `dev` branch will be merged into `main` after testing.
+
+`main` branch is the release branch. All content `main` appears in the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger)'s sample gallery.
+![The Office Add-ins Dev Kit sample gallery in VS Code.](assets/sample_gallery.png)
+
+## Testing
+
+TBD
+
+## Check in new sample into Sample Gallery
```

Comment:
Something missing in these instructions is the capability with VS Code. The project needs all the npm and webpack stuff to be consumed by the tool, correct? A list of those details will be helpful.